nodejs/week3/homework/warmup/index.js

```diff
@@ -1,6 +1,52 @@
 const express = require("express");
+const bodyParser = require("body-parser");
 const app = express();
 
+app.use(
+  bodyParser.urlencoded({
+    extended: true,
+  })
+);
```

Comment:
in latest express versions, you don't have to require the bodyParser, you can just do this:
```
app.use(express.urlencoded({ extended: true }));
```

Reply:
OK Thank you .